nabs/plan_stubs.py

```diff
@@ -74,15 +74,15 @@ def update_sample(sample_name, path, n_shots):
     """
     Update the current sample information after a run.
 
-    Updates the `status` values of each target in the sample,
-    from `False` to `True` to indicate that it is shot.
```

Comment:
These should have been OK as-is: 
`Built-in constants and keywords can be emphasized with backticks. An appropriate reference to python docs with be automatically generated (e.g. `True` or `assert`).` (https://pcdshub.github.io/style.html ref because I forget frequently)

Reply:
Hmm, bizarre, these definitely were making warnings (unless I messed up here?)

Reply:
https://travis-ci.com/github/pcdshub/nabs/jobs/482058753#L2916-L2917

Reply:
Hmm
```
/home/travis/build/pcdshub/nabs/docs/source/api.rst.rst:4: WARNING: autosummary: failed to import nabs.plans
/home/travis/build/pcdshub/nabs/docs/source/api.rst.rst:4: WARNING: autosummary: failed to import nabs.preprocessors
```

Reply:
Yes, that's one of the things fixed in this PR

Reply:
I was just wondering if it led to those messages

Reply:
Ahh, actually those were related to this fix: https://github.com/pcdshub/nabs/pull/39/commits/88557a54e52aba03c32ff5f26d56f1d792589c00

Reply:
Double-confirmed that those true/false errors pop back up if I undo the double backticks on true/false

Reply:
OK, looks like this is because we're missing "Python" in intersphinx:

(From pcdsdevices)
```
intersphinx_mapping = {
    'ophyd': ('https://blueskyproject.io/ophyd', None),
    'python': ('https://docs.python.org/3', None),
    'numpy': ('https://docs.scipy.org/doc/numpy', None)
}
```

Reply:
ahh, let me try adding that in

Reply:
better after adding this in